src/library_wget.js

```diff
@@ -232,9 +232,7 @@ var LibraryWget = {
   emscripten_async_wget2_abort__proxy: 'sync',
   emscripten_async_wget2_abort: (handle) => {
     var http = wget.wgetRequests[handle];
-    if (http) {
-      http.abort();
-    }
+    http?.abort();
```

Comment:
Maybe combine these into a single line and remove the curlies?

Reply:
It's an easy change but I'd prefer to leave unrelated cleanups separate.

This is already a pretty large PR and as mentioned above I already had some mistakes slipped in it before so I'd rather 1) have someone do another pass to check if conditions look good and 2) not extend it with more cleanups.